src/components/atoms/loadingWrapper/LoadingWrapper.tsx

```diff
@@ -1,42 +1,93 @@
-import React, { useState, HTMLAttributes } from 'react';
+import React, { useState, HTMLAttributes, useEffect } from 'react';
 import Loading from '../loading/Loading';
 import './loadingWrapper.scss';
 
 export interface LoadingWrapperInterface
   extends HTMLAttributes<HTMLDivElement> {
   data: any;
   animation?: boolean;
+  // option to make animation trigger after n milliseconds
+  startAfter?: number;
   animationTime?: number;
 }
 
 // TODO smoother rendereing of component
+// Render components after data is fetched
+// animationTime -> ensures that the loading animation is displayed for minimum "animationTime" milliseconds (only if animation is true)
+// startAfter -> if > 0 the render animation starts after "startAfter" milliseconds (only if animation is true)
 const LoadingWrapper: React.FC<LoadingWrapperInterface> = ({
   data,
-  animation = false,
+  animation = true,
   animationTime = 250,
+  startAfter = 0,
   children,
   ...rest
 }) => {
   // sets render to opposite to animation since animation true will need to explicit set
   // render for childeren to load and no animation will just render children when defined
+  const [isFetched, setIsFetched] = useState<boolean>(false);
+  const [animationRender, setAnimationRender] = useState<boolean>(false);
+  const [startAfterDone, setStartAfterDone] = useState<boolean>(
```

Comment:
I can't see any usage of `setStartAfterDone`(?). Should not use a `useState` if we don't need a setter, and we should just compute the variable with the prop then:

`const startAfterDone = startAfter === 0`

Reply:
good catch! it should have been `setStartAfterDone(true)` in `startAfterTimout`